src/schema/vacationRequest.ts

```diff
@@ -5,15 +5,15 @@ import { Type } from "@sinclair/typebox";
  */
 
 const vacationRequestSchema = Type.Object({
```

Comment:
If you decide to add status to the VacationRequest, wouldn't it also need to be here?